### PR TITLE
[NER] Support ernie and peoples_daily_ner.

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

example optimization
### PR changes

Others
### Description

[NER] Support ernie and peoples_daily_ner.

### CODE REVIEW

File: examples/information_extraction/msra_ner/train.py [hunk 1]
```diff
@@ -197,13 +220,21 @@ def do_train(args):
...
+                             label_num, "test")
+
                     paddle.save(model.state_dict(),
                                 os.path.join(args.output_dir,
                                              "model_%d.pdparams" % global_step))
 
 
 if __name__ == "__main__":
     args = parser.parse_args()
+    logger.info('{:20}:{}'.format("paddle commit id", paddle.version.commit))
```
Comment (joey12300) [inline examples/information_extraction/msra_ner/train.py]: 可以在args添加log level，默认不展示给用户看

**Review by joey12300 — CHANGES_REQUESTED** [review]

File: examples/information_extraction/msra_ner/train.py (hunk 1 above)
Comment (ZHUI) [reply examples/information_extraction/msra_ner/train.py]: 已删除 commit id 打印

Comment (ZHUI) [reply examples/information_extraction/msra_ner/train.py]: 已删除 commit id 打印

**Review by joey12300 — APPROVED** [review]
LGTM

**Pull request merged**